v03_pipeline/lib/tasks/dataproc/create_dataproc_cluster.py

```diff
@@ -0,0 +1,168 @@
+import time
+
+import hail as hl
+import luigi
+from google.cloud import dataproc_v1 as dataproc
+from pip._internal.operations import freeze as pip_freeze
+
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.model import Env, ReferenceGenome
+from v03_pipeline.lib.tasks.base.base_loading_pipeline_params import (
+    BaseLoadingPipelineParams,
+)
+
+CLUSTER_NAME_PREFIX = 'pipeline-runner'
+DEBIAN_IMAGE = '2.1.33-debian11'
+HAIL_VERSION = hl.version().split('-')[0]
+INSTANCE_TYPE = 'n1-highmem-8'
```

Comment:
should this be configurable?

Reply:
Good question... I left it un-configurable because we don't configure it, and in theory it's duplicated "scaling" config with the number of pre-emptibles.